core/test/financial_templates/PricelessPositionManager.js

```diff
@@ -467,5 +473,37 @@ contract("PricelessPositionManager", function(accounts) {
     await pricelessPositionManager.redeem({ rawValue: numTokens }, { from: sponsor });
     assert(await didContractThrow(pricelessPositionManager.redeem({ rawValue: numTokens }, { from: sponsor })));
   });
+
+  it("Basic fees", async function() {
+    // Set up position.
+    await collateral.approve(pricelessPositionManager.address, toWei("1000"), { from: sponsor });
+    await pricelessPositionManager.create({ rawValue: toWei("1") }, { rawValue: toWei("1") }, { from: sponsor });
+
+    // Set store fees to 1% per second.
+    await store.setFixedOracleFeePerSecond({ rawValue: toWei("0.01") });
+
+    // Move time in the contract forward by 1 second to capture a 1% fee.
+    const startTime = await pricelessPositionManager.getCurrentTime();
+    await pricelessPositionManager.setCurrentTime(startTime.addn(1));
+
+    // Determine the expected store balance by adding 1% of the sponsor balance to the starting store balance.
+    const expectedStoreBalance = (await collateral.balanceOf(store.address)).add(toBN(toWei("0.01")));
+
+    // Pay the fees, then check the collateral and the store balance.
+    await pricelessPositionManager.payFees();
+    let collateralAmount = await pricelessPositionManager.getCollateral(sponsor);
+    assert.equal(collateralAmount.rawValue.toString(), toWei("0.99"));
+    assert.equal((await collateral.balanceOf(store.address)).toString(), expectedStoreBalance);
+
+    // Ensure that fees are not applied to new collateral.
+    // TODO: value chosen specifically to avoid rounding errors -- see #873.
+    await pricelessPositionManager.deposit({ rawValue: toWei("99") }, { from: sponsor });
+    collateralAmount = await pricelessPositionManager.getCollateral(sponsor);
+    assert.equal(collateralAmount.rawValue.toString(), toWei("99.99"));
+
+    // Set the store fees back to 0 to prevent it from affecting other tests.
+    await store.setFixedOracleFeePerSecond({ rawValue: "0" });
+  });
+
   //TODO: check position is correctly deleted on settle expired
```

Comment:
Is this TODO still relevant

Reply:
I *think* so -- @chrismaree added it in one of his recent PRs.